src/components/Categories/ListDividers.tsx

```diff
@@ -17,18 +17,18 @@ const ListDividers = () => {
 	const navigate = useNavigate();
 
 	const hipHopHandler = () => {
-		navigate('/HipHop');
+		navigate('Genres/HipHop');
```

Comment:
widzisz, tutaj tez sie powtarza genres. Przykladowo, gdybys kiedys musial to zmienic na "genres" musialbys wykonac zmiane we wszystkich plikach